### ISSUE TITLE
Remove duplicate call to Textures.generate()

### ISSUE THREAD

**Issue by DragonDev1906** [DragonDev1906/Minecraft-Overviewer#32]
Upstream issue when trying to solve it: https://github.com/overviewer/Minecraft-Overviewer/issues/1731

**Comment by DragonDev1906** [comment]
The current version of the AssetLoader is not Pickelable. For testing the AssetLoader in textures.py was removed and recreated. This resulted in Preprocessing taking really long (or forever, see log below).

I'm not sure if this is directly linked to AssetLoader or to some other changes we made.

Commit: e6dc4c2b4e92652c2320aee80fb759c43a794e49
Branch: ubuntu-16-eof-error

How to recreate:

1. Install ubuntu 16.4
2. `apt install python3 python3-dev build-essential python3-venv xvfb libgles2-mesa-dev`
3. Maybe you need `apt install libgles2-mesa`, too
4. Create and setup venv: `python3 -m venv env`
5. `source env/bin/activate`
6. `pip install numpy Pillow moderngl`
7. Get sourcecode: `git clone https://github.com/DragonDev1906/Minecraft-Overviewer`
8. `cd Minecraft-Overviewer/`
9. `git branch ubuntu-16-eof-error origin/ubuntu-16-eof-error`
10. `python setup.py build`
11. Setup EGL: `export DISPLAY=:99.0`
12. `Xvfb :99 -screen 0 640x480x24 &`
13. Run: `python overviewer.py -c ../config --forcerender`

Config:
```
outputdir = "/home/user/overviewer/out"

worlds["Test12985"] = "/home/user/overviewer/Test12985"
worlds["Debug"] = "/home/user/overviewer/Debug"

renders["normalrender"] = {
    "world": "Test12985",
    "title": "Normal World",
}

renders["secondrender"] = {
    "world": "Debug",
    "title": "Debug World",
}

texturepath = "/home/user/overviewer/1.15.2.jar"
```

Log:
```
2020-04-01 11:26:28  No Element data found for the model block/bed
2020-04-01 11:26:28  No Element data found for the model block/yellow_shulker_box
2020-04-01 11:26:28  No Element data found for the model block/banner
2020-04-01 11:26:28  No Element data found for the model block/skull
2020-04-01 11:26:28  No Element data found for the model block/skull
2020-04-01 11:26:28  Preprocessing...
[(['5', '1', '1'], ['3', '2', '1'], ['1', '3', '1']), (['3', '4', '2'], ['8', '5', '2'], ['4', '6', '2']), (['7', '7', '3'], ['6', '8', '3'], ['8', '9', '3']), (['2', '10', '4'], ['8', '11', '4'], ['6', '12', '4']), (['1', '3', '5'], ['4', '13', '5'], ['2', '14', '5']), (['5', '15', '6'], ['2', '10', '6'], ['6', '12', '6']), (['5', '1', '1'], ['7', '16', '1'], ['3', '2', '1']), (['3', '4', '2'], ['7', '17', '2'], ['8', '5', '2']), (['7', '7', '3'], ['5', '1', '3'], ['6', '8', '3']), (['2', '10', '4'], ['4', '18', '4'], ['8', '11', '4']),
(['1', '3', '5'], ['3', '19', '5'], ['4', '13', '5']), (['5', '15', '6'], ['1', '20', '6'], ['2', '10', '6'])]
^Z
[2]+  Stopped                 python overviewer.py -c ../config --forcerender
```

**Comment by DragonDev1906** [comment]
Resolved in commit: 9cf35336b56c72ef197018af7b02e71a44d659f2
